projects/storefrontlib/src/cms-components/product/product-variants/variant-style-selector/variant-style-selector.component.ts

```diff
@@ -36,4 +41,24 @@ export class VariantStyleSelectorComponent {
       ? `${this.config.backend.occ.baseUrl}${qualifier.image.url}`
       : '';
   }
+
+  changeStyle(code: string): void {
+    if (code) {
+      // consider adding 'route' scope
```

Comment:
Is this comment still needed? Should we create a ticket for it or drop?

Reply:
ticket created: #6317

Reply:
and comment removed